### ISSUE TITLE
Problem: No trivial way to see baker baking status

### ISSUE THREAD

**Issue by clacke** [fractalide/tezos-stakepool#20]
The most important thing to see is: When will the next baking slot be, and do we have the funds for the bond?

But also:
 - How much is currently bound in bonds?
 - When will rewards come in?
 - How much is staked?
 - What is our staking limit, based on the current balance, bonds and baking frequency?

**Comment by sjmackenzie** [comment]
so @qknight nixcloud monitoring service pretty much details if the node is alive or not. Though you can extract custom information. It would be super cool if @qknight could expose this custom data in two ways: 1 as an api so we can query it to represent data how we want or 2 a visually pleasing badge that can be embedded where appropriate. We need to evolve thin solution.

Let's determine exactly what data our users need to see then put @qknight 's monitoring systems to the test. After launch the tezos stake pool we can do incremental steps in this direction as the ICO will take priority shortly.

**Comment by clacke** [comment]
Yeah, that kind of monitoring is really a separate issue though. The data I'm talking about here is observable on the chain, I just separated it from #21 because #21 is not necessary if you don't have outsiders staking with you, but the stuff listed here is necessary even if you're just staking for yourself, if you don't want to miss any bakes.

**Comment by sjmackenzie** [comment]
Ah yeah, it's pointless that data going through nixcloud servers

**Comment by qknight** [comment]
@sjmackenzie not sure what you guys are actually monitoring but we could support both: 

* badges
* live-feed with status updates (no REST but a websocket which pushes data)

**Comment by sjmackenzie** [comment]
@qknight yeah the data we're speaking about pertains to displaying data the service generates. nixcloud's responsibility is 3rd party monitoring of services we create, so in this case normal uptime badges will be good for our users. Other domain specific data the service generates should be exposed via websockets/other means for our users to see on the website.

Although, from another perspective: we're running on nixcloud infrastructure anyway... maybe we can hack the software so that it uses nixcloud.monitoring but doesn't shunt that user specific data to nixcloud's dashboard but puts it on our own end point?

**Comment by sjmackenzie** [comment]
Claes would it be better to build a separate styx website frontend for this stake pool so others can easily pull it in and integrate directly with nixcloud infrastructure?

**Comment by clacke** [comment]
Yes, separate web service for this is definitely what I'm thinking.

**Comment by sjmackenzie** [comment]
Probably wisest, then that delegation address on the mockup should be a link through to the webservice. (fyi @dapyh17d4). We can reuse much of the same logic for the cardano stake pool too.

**Comment by clacke** [comment]
Right, good idea. And yes, I think all the concerns we are addressing while setting up the Tezos pool will help us know what to think about, and maybe even enable some reuse, when we stake on Cardano.

**Comment by clacke** [comment]
Most of the relevant information is conveniently available in the Tezos node API:

https://tezos.gitlab.io/betanet/api/rpc.html#get-block-id-context-delegates-pkh

**Comment by clacke** [comment]
Don't forget endorsing writes and the bonds for that.

**Comment by clacke** [comment]
https://github.com/FRAMISES/TezosCheatSheet/blob/master/README.md#snapshots says you should be able to get the chosen snapshot by querying `/chains/main/blocks/head/context/raw/json/rolls/owner/snapshot/<cycle>`.

https://github.com/dakk/tezpool/blob/af90dfab6fbeacd119abcca0543486676356ec36/tezpool.py#L76 used to use that technique, but has since then abandoned it.

When I try it I get:

```
$ clientmain rpc get /chains/main/blocks/head/context/raw/json/rolls/owner/snapshot/78
[]
```

As the call wasn't documented, I guess it may have been silently deprecated.

Will go with the last block of the relevant cycle for now, later maybe the average over the whole cycle (or the average over the 16 snapshots over the cycle, perhaps).
